components/authz-service/storage/v2/conditionattribute.go

```diff
@@ -0,0 +1,76 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ConditionAttribute is an enum of attributes a project rule condition can be.
+type ConditionAttribute int
+
+const (
+	Organization ConditionAttribute = iota
```

Comment:
Could we alphabetize these? 😬 🤓

Reply:
I updated them to be alphabetic, but in general you don't wanna change the order of enums because the order matters, so if we ever add another one, it'll likely be un-alphabetized anyway.